### PR TITLE
Readjusting brightness (Pinecil v2)

### LINKED ISSUE #1620: Screen Brightness  collapse 1-10 into  levels 1-5  (eliminate level 1 as it makes some sceens BLACK)
**Is your feature request related to a problem? Please describe.**

1. Pinecil V2:  some people report  that when they bring screen brightness to level "1" the screen goes black instead of just low dim like expected. one of the people is builder555 (PineSAM). we have seen this on several models now brand new from factory.

2.  one person who had this happen  got a little scared that their new V2 was broken when they were  going through settings menu changing the brightness,  and instead of  pressing (+)  a couple times to get back to a higher brightness  that is visible,  started Rapidly and Randomly pressing  +/-   (fear happens) and then  got stuck and could not see where they were at in the screen hierarchy. 
 Flashing new firmware did not help them since flashing  keeps all the same settings and we don't have a "blank.bin" to flash and clear the rom yet. this would help clear rom, then a new firmware flash would  restore factory defaults.

Temporary Solution:
3. I had them flash a BLE version of IronOS, and then with **PineSAM** app they were able to adjust their Screen back to 4 so they could see it again. 

**Solution Proposed**

1. Could you let Pine Store know that whatever is going on hardware-wise, level 1 screen brightness is **BLACK** on some of these and people think they have a broken pinecil and since reflashing doesn't fix it, they get scared.
2.  Thanos suggested that that using a different kind of **Capacitor** under the screen area on the PCB could  fix this.  best to talk to thanos on this, but he had a  specific idea in mind of what kind they need to change to in order for the screen to not go black at  level "1" and overhall these new caps he suggested would improve quality.
3.  Meanwhile, Is there is anything that can be done in firmware to  make "1"  not get so low that it's black screeen?  

**Describe the solution you'd like**
In addition, I propose  we only have levels 1-5  on brightness **(collapsing the 10 levels into 5)** since on Pinecil V2, people can not see any difference in many of the middle levels.
we tested with several people and there is no visual difference from levels  2-8, therefore  could firmware be changed so that there are only  5 total levels since all the other levels show no difference in brightness?
I  can see "1" and "10" are different. realistically, even with all the room lights off, the 2-8 had no change to the eye. and "9" gets slightly brighter.

- Whatever 1-2 is , eliminate "1" level since it's too low for some people to see and screen is black. make  the new "1" whatever old 2 was.
- 1-2 (specifically 2) becomes 1
- 3-4  (specifically 4) becomes  2
- 5-6 (specifically 6) becomes 3
- 7-8 (sepcifically 8) becomes 4
- 9-10 (sepcifically 10) becomes 5

**VIDEO** example of normal Pinecils.
I'm using 3 different pinecils all at the same time in this video so it's not a great control.  it should have been one single pinecil filmed at levels   1, 5, and 10

Bottom = **1**, middle = **5**,   top one = **10**
https://user-images.githubusercontent.com/97197236/224641160-2c88e677-77ea-4812-bb9f-609be905bb6b.mp4

### PR DESCRIPTION
**Pull request by discip**
Resolves #1620

### CODE REVIEW

**Comment by River-Mochi** [comment]
3 people tested this on Pinecil V2.  looks good.

File: source/Core/Src/settingsGUI.cpp [hunk 1]
```diff
@@ -616,7 +616,7 @@ static void displayAnimationLoop(void) { OLED::drawCheckbox(getSettingValue(Sett
 
 static void displayBrightnessLevel(void) {
 
-  OLED::printNumber((getSettingValue(SettingsOptions::OLEDBrightness) / 11 + 1), 2, FontStyle::LARGE);
+  OLED::printNumber((getSettingValue(SettingsOptions::OLEDBrightness) / 25 + 1), 2, FontStyle::LARGE);
```
Comment (Ralim) [inline source/Core/Src/settingsGUI.cpp:619]: I think we should extract this magic 25 as a contact of like BRUGHTNESS_STEP ?

**Review by Ralim — CHANGES_REQUESTED** [review]

File: source/Core/Src/settingsGUI.cpp (hunk 1 above)
Comment (discip) [reply source/Core/Src/settingsGUI.cpp]: Does this meet your request?

Comment (discip) [reply source/Core/Src/settingsGUI.cpp]: Does this match your request?

Comment (discip) [reply source/Core/Src/settingsGUI.cpp]: Does this match your request? 😊

**Review by Ralim — APPROVED** [review]

**Pull request merged**